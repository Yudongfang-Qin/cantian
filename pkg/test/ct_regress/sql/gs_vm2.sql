
--Do not change expectations easily; there may be a virtual memory leak
select ID,TOTAL_VIRTUAL,FREE_VIRTUAL,PAGE_SIZE,TOTAL_PAGES,FREE_PAGES from v$temp_pool;
select count(1) from v$vm_func_stack;
