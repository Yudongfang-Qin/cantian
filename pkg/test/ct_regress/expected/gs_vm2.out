

SQL> 
SQL> --Do not change expectations easily; there may be a virtual memory leak
SQL> select ID,TOTAL_VIRTUAL,FREE_VIRTUAL,PAGE_SIZE,TOTAL_PAGES,FREE_PAGES from v$temp_pool;

ID           TOTAL_VIRTUAL FREE_VIRTUAL PAGE_SIZE    TOTAL_PAGES  FREE_PAGES  
------------ ------------- ------------ ------------ ------------ ------------ 
0            3276          3273         131072       2047         2044
  
1 rows fetched.                                                             

SQL> select count(1) from v$vm_func_stack;

COUNT(1)            
--------------------
4                   

1 rows fetched.

SQL> 


